src/codeu/chat/client/commandline/Chat.java

```diff
@@ -46,11 +46,19 @@ public Chat(Context context) {
   // the system wants to exit, the function will return false.
   //
   public boolean handleCommand(String line) {
-
+/*
```

Comment:
You'll probably want to remove any commented code before trying to merge in your changes here.

The nice thing about version control systems (including Git) is that you don't need to leave things commented-out in order to save them. You can do a commit with the code that you want to disable, then delete it completely, then get it back from version control later if you need it.